R/qenv-eval_code.R

```diff
@@ -9,6 +9,7 @@
 #' @param code (`character`, `language` or `expression`) code to evaluate.
 #' It is possible to preserve original formatting of the `code` by providing a `character` or an
 #' `expression` being a result of `parse(keep.source = TRUE)`.
+#' @param ... ([`dots`]) additional arguments passed to future methods.
```

Comment:
`?eval_code` doesn't show this in docs. `within.qenv` shares a documentation with `eval_code` and `...` is overriden by docs of `within`. I'd separate both documentation now